web/src/components/gnd-project-editor/index.js

```diff
@@ -54,10 +53,10 @@ const styles = (theme: Object) => ({
 // TODO: Define Project object type.
 type Props = {
   classes: Object,
+  close: Function,
```

Comment:
Is `type` a feature of Flow or React? How does it relate to `PropTypes`?

Reply:
To be honest I've never seen this type of declaring prop types before. 
I used to just use "Component.propTypes ="

but now when you say it googling leads to flow pages indeed https://flow.org/en/docs/react/components/

Do you want me to change it?

Reply:
It sounds like the linter then was checking for Flow types, not React PropTypes? I think using Flow for props type checking is better because it will now also now be able to do static type checking in code that references these props, so this looks right. Thanks!